### PR TITLE
Add new managed k8s page

### LINKED ISSUE #2333: Build kubernetes/managed

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added a new managed kubernetes section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes/managed](http://0.0.0.0:8001/kubernetes/managed)
- Check the page content against the [copydoc](https://docs.google.com/document/d/1viNQgt99D2_b873Yd98iNT5vEzIbPQFgNI5XBFnszHk/edit#)

## Issue / Card

Fixes #2333

### CODE REVIEW

**Comment by frankiegranato** [comment]
Looking good. Two minor requests in the copydoc.

File: templates/kubernetes/managed.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    </div>
+  </section>
+
+  <section class="p-strip is-bordered">
+    <div class="row">
+      <h2>Features</h2>
+      <div class="col-8">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">As Kubernetes evolves, your private cloud can be upgraded  on request   without down-time.</li>
+          <li class="p-list__item is-ticked">When you want to scale your cluster, just call to let us know there are additional machines in   the DC or virtualization environment and we’ll expand your Kubernetes with zero down time.</li>
```
Comment (grahambancroft) [inline templates/kubernetes/managed.html]: Should be 'virtualisation', I've changed it in the copy doc

**Comment by spencerbygraves** [comment]
I think an image for the top section would be good. I'll try to come up with something.

**Comment by spencerbygraves** [comment]
@frankiegranato Could we use this? It looks like the official image.

image

**Comment by frankiegranato** [comment]
You need to change the heading of the pricing table (I've add a -1 label so you know I've reviewed it - not to be mean!).

**Comment by deadlight** [comment]
@frankiegranato that's exactly what the label is for! I'll make these small changes.

**Comment by deadlight** [comment]
@frankiegranato @spencerbygraves ready for re-review (I've rebuilt the demo with the latest version)

**Comment by frankiegranato** [comment]
@deadlight Sorry Karl. We need to remove this logo from the page. We are not an official "managed service provider" so its misleading to show the logo. Nice idea though @spencerbygraves

**Comment by frankiegranato** [comment]
@spencerbygraves Peter suggested you try to add the K8s logo into this: https://assets.ubuntu.com/v1/c5838368-openstack-cloud.png
I think he means a bit like how we do so with the openstack logo. 
Can you do this for us? If you don't have time though then I understand. 
Fyi @deadlight

**Comment by spencerbygraves** [comment]
@deadlight I've attached an SVG as suggested above. 
[K8s cloud.svg.zip](https://github.com/canonical-websites/www.ubuntu.com/files/1453466/K8s.cloud.svg.zip)

image

**Comment by grahambancroft** [comment]
@frankiegranato @spencerbygraves I've added the new logo, this is now ready for re review

File: templates/kubernetes/managed.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+{% block title %}Managed Kubernetes | Kubernetes{% endblock %}
+{% block meta_description %}With managed kubernetes, Canonical will build, support and manage your cluster for only $4 per node per day.{% endblock meta_description %}
+{% block extra_body_class %}kubernetes{% endblock %}
+
+
+{% block content %}
+
+  <section class="p-strip is-deep is-bordered">
```
Comment (pmahnke) [inline templates/kubernetes/managed.html]: I think this should be `p-strip--accent is-deep` like on [cloud/managed-cloud](http://0.0.0.0:8001/cloud/managed-cloud)

File: templates/kubernetes/managed.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-7">
+        <h1>Managed Kubernetes</h1>
+        <p>Focus on your business while Canonical takes care of building and running your Kubernetes infrastructure.
+        Canonical will set-up and operate your Kubernetes cluster for you, with in-house experts on-hand at all times to stand-up and scale your cluster. And, when you&rsquo;re ready, we will hand over operational control.</p>
+        <ul>
+          <li>Fully managed Kubernetes on your bare metal, VMWare, OpenStack or any public cloud</li>
+          <li>Built in two weeks using a proven reference architecture with our <a href="/kubernetes">Kubernetes consulting packages</a></li>
```
Comment (pmahnke) [inline templates/kubernetes/managed.html]: and the link needs a class="p-link--inverted"

Comment (pmahnke) [reply templates/kubernetes/managed.html]:
```
<section class="p-strip--accent is-deep">
    <div class="row u-equal-height u-vertically-center">
      <div class="col-7">
        <h1>Managed Kubernetes</h1>
        <p>Focus on your business while Canonical takes care of building and running your Kubernetes infrastructure.
        Canonical will set-up and operate your Kubernetes cluster for you, with in-house experts on-hand at all times to stand-up and scale your cluster. And, when you’re ready, we will hand over operational control.</p>
        <ul>
          <li>Fully managed Kubernetes on your bare metal, VMWare, OpenStack or any public cloud</li>
          <li>Built in two weeks using a proven reference architecture with our <a class="p-link--inverted" href="/kubernetes">Kubernetes consulting packages</a></li>
          <li>Customise the architecture in iterations based on workloads</li>
          <li>Enterprise class SLA for the infrastructure, upgrades included</li>
          <li>Monitoring and log management integration</li>
          <li>When you’re ready, we will hand over the keys</li>
        </ul>
        <p><a href="/containers/contact-us?product=containers-managed-kubernetes" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Schedule a Kubernetes demo', 'eventLabel' : 'Schedule a Kubernetes demo - Top CTA' : undefined });" class="p-button--positive">Schedule a Kubernetes demo</a></p>
      </div>
      <div class="col-5 u-align--center u-hidden--small">
        <img src="https://assets.ubuntu.com/v1/35b71a5e-K8s-cloud.svg" alt="Kubernetes Managed Service Provider">
      </div>
    </div>
  </section>
```

image

**Pull request merged**